README.md

```diff
@@ -25,129 +25,67 @@ Setup of tools that allow you to play with & test out the UI with ease.
 
-### 8. Get easycut-smartbench from GitHub into Eclipse 
+Then, go to `Git > Clone` and copy and paste the link to the GitHub repository in the URL section and select where you'd like the repository to get stored **locally**. Make sure you remember this location. For more details click [here](https://www.jetbrains.com/help/pycharm/manage-projects-hosted-on-github.html)
 
-In Eclipse, go to the toolbar and select Window > Show View > Other. 
+If the code shows a lot of errors, don't worry, it's expected at this stage.
 
-In the pop-up, expand the "Git" folder, and click on "Git Repositories". Click OK. 
+### 4. Select your interpreter in your IDE
 
-Then, in the Eclipse toolbar go to File > Import. 
+Once again, this step will vary depending on your IDE, so please consult your IDE's website if you use something other than PyCharm.
 
-In the pop-up, select Projects from Git, and then Next. 
+In PyCharm, when you cloned the repository, you should have gotten a notification asking you to select the interpreter. If you did, click on it and select `Python 2.7` from the dropdown list.
 
-Click Clone from URI. 
+If you didn't get the notification, navigate to `File > Settings > Project: <project name> > Python Interpreter` and select `Python 2.7` as your interpreter.
 
-Paste the following into the URI box: 
+If `Python 2.7` does not show up in the list, go to `Add Interpreter > Add Local Interpreter` and navigate to the location where you installed python and select the `python.exe` file. If you installed in the default location, this should be `C:/Python27/python.exe`.
 
-https://github.com/YetiTool/easycut-smartbench.git
+For additional information, click [here](https://www.jetbrains.com/help/pycharm/configuring-python-interpreter.html#add-existing-interpreter)
 
-The other information should auto-fill, so click Next. Click Next again. Choose a directory to install it in, and click Next again. 
+### 5. Install dependencies
 
-Select "Import existing projects", click Next, and then click Finish. 
+Now we can put aside the IDE open Command Prompt to install the dependencies.
 
-That should be you done! easycut-smartbench will show up under Git Repositorie.
+Navigate to the folder where you saved the repository. You can do this using `cd <path to repository>`. NOTE: if the path is on a separate partition, for example `D:`, you will need to enter `D:` in the command prompt before you can use `cd <path to repository>`.
 
-Make sure you can also see Package Explorer in Eclipse - if you can't just go to Window > Show View > Package Explorer. 
+You should now be in the same repository as the `requirements.txt` file. In this case, you can simply enter `pip install -r requirements.txt` and wait for pip to install all the dependencies. You may need to occasionally type `Y` or `yes` to agree to installing the dependencies.
```

Comment:
This is lovely and simple - like it a lot!

A note on the requirements file - I can see a few packages that I don't recognize, so I don't know whether everything in the file is actually necessary?

Did you generate the file while you were in a virtual environment? If not then it might have picked up a few packages you had locally that aren't actually needed :)

Can this requirements file be used on the pi as well, or is it just windows atm?